### PR TITLE
[ATMOSPHERE-385] chore: sync vendor charts

### PR DESCRIPTION
**Pull request by okozachenko1203**

### CODE REVIEW

File: charts/libvirt/templates/daemonset-libvirt.yaml [hunk 1]
```diff
@@ -299,8 +299,6 @@ spec:
               {{- if or ( gt .Capabilities.KubeVersion.Major "1" ) ( ge .Capabilities.KubeVersion.Minor "10" ) }}
               mountPropagation: Bidirectional
               {{- end }}
-          args:
-            - "--libvirt.nova"
```
Comment (mnaser) [inline charts/libvirt/templates/daemonset-libvirt.yaml:303]: I think we need a patch to keep this.

File: charts/ironic/values.yaml [hunk 2]
```diff
@@ -406,7 +406,7 @@ endpoints:
         user_domain_name: service
         project_domain_name: service
       ironic:
-        role: admin,service
```
Comment (mnaser) [inline charts/ironic/values.yaml:409]: We need a patch to add this.

File: charts/ironic/templates/bin/_ironic-conductor-pxe-init.sh.tpl [hunk 3]
```diff
@@ -43,7 +43,7 @@ fi
 mkdir -p /var/lib/openstack-helm/tftpboot
 mkdir -p /var/lib/openstack-helm/tftpboot/master_images
 
-for FILE in undionly.kpxe ipxe.efi pxelinux.0 snponly.efi; do
```
Comment (mnaser) [inline charts/ironic/templates/bin/_ironic-conductor-pxe-init.sh.tpl:46]: We need a patch to add this.

**Review by mnaser — CHANGES_REQUESTED** [review]
Small change.

File: charts/ironic/templates/bin/_ironic-conductor-pxe-init.sh.tpl (hunk 3 above)
Comment (yaguangtang) [reply charts/ironic/templates/bin/_ironic-conductor-pxe-init.sh.tpl:46]: moved to a local patch

File: charts/ironic/values.yaml (hunk 2 above)
Comment (yaguangtang) [reply charts/ironic/values.yaml:409]: moved to a local patch

File: charts/libvirt/templates/daemonset-libvirt.yaml (hunk 1 above)
Comment (yaguangtang) [reply charts/libvirt/templates/daemonset-libvirt.yaml:303]: moved to a local patch

File: charts/ironic/values.yaml (hunk 2 above)
Comment (okozachenko1203) [reply charts/ironic/values.yaml:409]: https://review.opendev.org/c/openstack/openstack-helm/+/925931

File: charts/libvirt/templates/daemonset-libvirt.yaml (hunk 1 above)
Comment (okozachenko1203) [reply charts/libvirt/templates/daemonset-libvirt.yaml:303]: https://review.opendev.org/c/openstack/openstack-helm-infra/+/925930

**Review by mnaser — APPROVED** [review]

**Pull request merged**